### ISSUE TITLE
Alt text on image uploads

### ISSUE THREAD

**Issue by henrywright** [Automattic/liveblog#127]
I've been reading up on alt text and accessibility. There doesn't seem to be a way of adding alt text to image uploads in liveblog entries?

Ref:
http://www.w3.org/standards/webdesign/accessibility

> If alt text isn't provided for images, the image information is inaccessible, for example, to people who cannot see and use a screen reader that reads aloud the information on a page, including the alt text for the visual image.
> 
> When equivalent alt text is provided, the information is available to everyone to people who are blind, as well as to people who turned off images on their mobile phone to lower bandwidth charges, people in a rural area with low bandwidth who turned off images to speed download, and others. It's also available to technologies that cannot see the image, such as search engines.
